pkg/watchdog/watchdog.go

```diff
@@ -77,9 +77,7 @@ func WatchdogFileUpdate(watchdogFile string, uid string) error {
 	if err != nil {
 		return err
 	}
-	f.Close()
-
-	return nil
+	return f.Close()
```

Comment:
I think that function had an error before already. After `os.Create` returned without an error we already need the `defer f.close()`.

Reply:
Done